release-notes/release-notes52.md

```diff
@@ -223,6 +229,97 @@ For information on the `toolbar_mode` option, see: [User interface options - `to
 
 This section describes issues that users of {{site.productname}} 5.2 may encounter, as well as possible workarounds for these issues.
 
+**Outline**
+
+* [Translations missing for new user interface options](#translationsmissingfornewuserinterfaceoptions)
+* [Floating toolbar disconnects from the toolbar in inline mode](#floatingtoolbardisconnectsfromthetoolbarininlinemode)
+* [Microsoft Internet Explorer will incorrectly auto focus the editor when TinyMCE is initialized](#microsoftinternetexplorerwillincorrectlyautofocustheeditorwhentinymceisinitialized)
+* [Incorrect positioning of inline dialogs when using sticky toolbars and the bottom toolbar](#incorrectpositioningofinlinedialogswhenusingstickytoolbarsandthebottomtoolbar)
+
+### Translations missing for new user interface options
+
+#### Issue
+
+New user interface settings do not have translations to other supported languages at this time. This includes text related to:
+
+- The new accessibility settings.
+- The new checklist menu item.
+
+#### Workaround
+
+No known workaround at the time of release.
+
+### Floating toolbar disconnects from the toolbar in inline mode
+
+#### Issue
+
+> **Note**: This issue will be resolved with the release of {{site.productname}} 5.2.1.
+
+If the editor is set to `inline: true`, and `toolbar_drawer` or `toolbar_mode` is set to `'floating'`, such as:
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  inline: true,
+  toolbar_drawer: 'floating'
+});
+```
+
+or
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  inline: true,
+  toolbar_mode: 'floating'
+});
+```
+
+The floating component of the toolbar will disconnect from the primary toolbar when page or editor is scrolled using a mouse or a scrolling touch gesture.
+
+#### Workaround
+
+Set `toolbar_drawer` or `toolbar_mode` to an alternative mode until the issue is resolved. For a list of alternative toolbar modes, see: [User interface options - `toolbar_mode`]({{site.baseurl}}/configure/editor-appearance/#toolbar_mode).
+
+### Microsoft Internet Explorer will incorrectly auto focus the editor when TinyMCE is initialized
+
+#### Issue
+
+> **Note**: This issue will be resolved with the release of {{site.productname}} 5.2.1.
+
+When a webpage containing {{site.productname}} is loaded on Microsoft Internet Explorer, the user's cursor will automatically focus on the editor when the editor is loaded.
```

Comment:
`cursor` isn't the correct term here (that normally refers to the mouse cursor). It's the editor component will steal focus from whatever else on the page previously had focus.

So as an example, if there was an input field for first/last name on a "contact us" form and TinyMCE was initialised, then instead of it being on the first name field it'd always end up on the editor instead.

Reply:
>...the web-browser focus will automatically move to the editor when the editor is loaded.